_posts/en/newsletters/2020-01-08-newsletter.md

```diff
@@ -0,0 +1,96 @@
+  summary, unequal inputs/outputs coinjoins [knapsack][] is the best we
+  have at the moment ([but] it is not as effective as equal-outputs
+  transactions)."
+
+- **Encoded descriptors:** Chris Belcher [asked][belcher descriptors]
+  for feedback from the Bitcoin-Dev mailing list about base64 encoding
+  [output script descriptors][topic descriptors] so that they're easier
+  to copy and paste (and also so that regular users aren't exposed to
+  their code-like syntax).  At least one reply was opposed to the idea
+  and other replies which did support the idea each proposed a different
+  encoding format.  The discussion to date did not come to any
+  clear conclusion.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3351][] extends the `invoice` RPC with a new
+  `exposeprivatechannels` parameter that allows the user to request
+  the addition of route hints for private channels to a generated [BOLT11][]
+  invoice.  The user may optionally specify which channels they want to
+  advertise in the invoice, including both public and private channels.
+
+- [LND #3647][] switches from using base64 to hex for the display of
+  binary data in the `listinvoices` RPC.  This is a breaking API change for users of any
+  of the updated fields.
+
+- [LND #3814][] allows the UTXO sweeper to add wallet inputs to a
+  sweep transaction in order to ensure its output meets the dust limit.
+  This is designed to help support the proposed anchor outputs feature
+  (see [Newsletter #70][news70 anchor]) which will need to add inputs to
+  a transaction in order to be able to spend low-value UTXOs.
+
+{% include linkers/issues.md issues="3647,3814,3351" %}
+[wuille slides]: https://prezi.com/view/AlXd19INd3isgt3SvW8g/
+[wuille suggestion]: http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-12-17-19.01.log.html#l-8
+[zmn post]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-December/002375.html
+[ficsor non-equal]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-December/017538.html
+[cashfusion]: https://github.com/cashshuffle/spec/blob/master/CASHFUSION.md
+[knapsack]: https://www.comsys.rwth-aachen.de/fileadmin/papers/2017/2017-maurer-trustcom-coinjoin.pdf
+[coinjoin sudoku]: http://www.coinjoinsudoku.com/
+[payment batching]: https://github.com/bitcoinops/scaling-book/blob/master/x.payment_batching/payment_batching.md
+[belcher descriptors]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-December/017529.html
+[news70 anchor]: /en/newsletters/2019/10/30/#ln-simplified-commitments
+[taproot meeting]: http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-12-17-19.01.html
```

Comment:
perhaps this link instead? http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-12-17-19.01.log.html

Reply:
We link to a line in the log a few sentences later, so I made this a link to the overview.  I admit that this overview is particularly useful, but I think it's better than linking to the same file twice.